InterGalacticCurrencyConvertor/src/main/java/DataProcessResult.java

```diff
@@ -0,0 +1,44 @@
+package main.java;
+
+public class DataProcessResult{
+	private String mOutput;
+	private Boolean mSuccess;
+
+	public DataProcessResult () {
```

Comment:
This default c'tor is not needed. It just assigns the member variables to their default values.

Reply:
updated